region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -195,11 +196,17 @@
         {
           "id": 4,
           "note": "This is a one-way link because the way back goes 4 -> 2."
-        }
+        },
+        {"id": 6}
       ]
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
```

Comment:
4 needs one
I've seen 5 CFd before.
3 is safe if you stand by the door
It would be useful to have one at 2 because some of the 6 strats cost a bunch of health.  You would go to the floating platform to avoid the namihe, but have to move sorta quick.  But then most of the [2,4] strats won't work without a reload.